### ISSUE TITLE
sampleScatterMatrix

### ISSUE THREAD

**Issue by hscotthinton** [opencobra/cobratoolbox#110]
The sampleSCatterMatrix function does not plot the correlation coefficients. If lines 143-144  have the comment symbol removed, it does plot the coefficients.

**Comment by rmtfleming** [comment]
submit a pull request with the fix?

On 13 February 2016 at 17:01, hscotthinton <notifications@github.com> wrote:

> The sampleSCatterMatrix function does not plot the correlation
> coefficients. If lines 143-144 have the comment symbol removed, it does
> plot the coefficients.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/opencobra/cobratoolbox/issues/110>.
>



-- 
--
Mr. Ronan MT Fleming B.V.M.S. Dip. Math. Ph.D.
----------------------------------------------------------------------------
Senior research associate (EN) == Chercheur (FR),
Principal investigator,
Systems Biochemistry Group,
wwwen.uni.lu/lcsb/research/systems_biochemistry
Luxembourg Centre for Systems Biomedicine,
University of Luxembourg,
Campus Belval,
6, avenue du Swing,
L-4367 Belvaux.
&
National Centre of Excellence in Research on Parkinson’s disease
www.parkinson.lu
&
Adjunct Assistant Professor,
Division of Analytical Biosciences,
Leiden Academic Centre for Drug Research,
Faculty of Science,
University of Leiden.
http://analyticalbiosciences.leidenuniv.nl
----------------------------------------------------------------------------
Mobile:  +352 621 175 112
Office: +352 466 644 5528
Skype: ronan.fleming
----------------------------------------------------------------------------
(This message is confidential and may contain privileged information. It is
intended for the named recipient only. If you receive it in error please
notify me and permanently delete the original message and any copies.)

**Comment by hscotthinton** [comment]
I am not sure what you are asking me to do? I do not think I understand this code well enough to submit a final fix. I noticed that if I uncommented a couple of lines then the coefficients were printed on the plot. I do not know why they were commented out in the first place. Perhaps there is another problem caused by that code which is the reason they were commented out. I thought it would be helpful for someone who really understands that function to help them troubleshoot the problem.
